client/src/components/MealCard.js

```diff
@@ -0,0 +1,127 @@
+// WILL MOVE ADD TO CART BUTTON ONCE CHEF PROFILE PAGE IS CREATED
+import React, { useContext } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+    Card,
+    Divider,
+    Avatar,
+    CardActions,
+    CardContent,
+    CardMedia,
+    Button,
+    Typography,
+    Grid,
+} from "@material-ui/core";
+import { theme } from "../themes/theme";
+import { CartContext } from "../contexts/cart/CartContextProvider";
+import { dollarFormatter } from "../lib/utils";
+
+const useStyles = makeStyles({
+    root: {
+        maxWidth: 225,
+        borderRadius: "0px",
+        boxShadow: "0px 0px 10px 5px rgba(7,7,7,0.05)",
+        margin: theme.spacing(3),
+    },
+    small: {
+        marginTop: theme.spacing(0.5),
+        marginBottom: theme.spacing(0.5),
+        marginLeft: theme.spacing(1),
+        width: theme.spacing(4),
+        height: theme.spacing(4),
+    },
+    subtitle2: {
+        fontSize: ".3rem",
+        color: theme.palette.grey[600],
+    },
+    subtitle1: {
+        fontWeight: "bold",
+        fontSize: ".7rem",
+    },
+    subtitle: {
+        fontWeight: "bold",
+    },
+});
+
+function MealCard(props) {
+    const classes = useStyles();
+    const { chef, addToCart } = useContext(CartContext);
+    const { mealPic, title, price, chefName, chefPic, location, id, chefId } = props;
+
+    const addMeal = (e) => {
+        e.preventDefault();
+        const id = parseFloat(e.currentTarget.value);
+        if (chef && chefId !== chef) {
+            alert(
+                `Your cart currently contains meals from another chef. You can only checkout with meals from one chef`
+            );
+        } else {
+            const meal = { id, mealPic, title, price, chefName, chefId };
+            addToCart(meal, id);
+        }
+    };
+
+    return (
+        <div>
+            <Card className={classes.root}>
+                <CardMedia
+                    component="img"
+                    alt="meal1"
+                    height="150"
+                    image={mealPic}
+                    title="meal1"
```

Comment:
If we have a whole array of mapped meals, we probably don't want them all to be "meal1". Can we use title for both alt and title here?